mypy/binder.py

```diff
@@ -189,19 +199,13 @@ def pop_frame(self, can_skip: bool, fall_through: int) -> Frame:
 
         return result
 
-    def get_declaration(self, expr: Node) -> Type:
-        if isinstance(expr, RefExpr) and isinstance(expr.node, Var):
-            type = expr.node.type
-            if isinstance(type, PartialType):
-                return None
-            return type
-        else:
-            return None
-
     def assign_type(self, expr: Expression,
                     type: Type,
                     declared_type: Type,
                     restrict_any: bool = False) -> None:
+        # TODO: replace with isinstance(expr, BindableTypes)
```

Comment:
Can you file an issue for that regression? What does reveal_type(BindableTypes) show at this point?